src/app/azureResources/components/azureResourcesContainer.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
```

Comment:
Learning from Ashish's comment for new FC patterns, we would adopt the pattern that new separate container files is needed but simply put it inside of the FC view.